src/factories/animation.ts

```diff
@@ -0,0 +1,18 @@
+import gsap from 'gsap'
+import { waitForConfig } from '@/objects/config'
+import { pauseCulling, resumeCulling } from '@/objects/culling'
+
+export async function animate(el: gsap.TweenTarget, vars: gsap.TweenVars, skipAnimation?: boolean): Promise<gsap.core.Tween> {
+  const config = await waitForConfig()
+  const duration = skipAnimation ? 0 : config.animationDuration / 1000
+
+  pauseCulling()
+
+  return gsap.to(el, {
+    ...vars,
+    duration,
+    ease: 'power1.out',
+  }).then(() => {
+    resumeCulling()
+  })
```

Comment:
Couldn't we just await to `gsap.to` and then call cull? 

And just make this return `Promise<void>`. I don't think returning the tween is actually useful.

Reply:
not relevant anymore